.travis.yml

```diff
@@ -48,9 +48,11 @@ install:
     fi
   - if [ "$TRAVIS_OS_NAME" == "linux" ] && [ "$CXX" = "g++" ]; then export CXX="g++-6"; fi
   - if [ "$TRAVIS_OS_NAME" == "linux" ] && [ "$CXX" == "clang++" ]; then export CXX="clang++-3.8"; fi
+  - pip install -r travis/python_reqs.txt
 
 script:
-  - bash .travis.sh
+  - bash travis/check_style.sh
+  - bash travis/travis_main.sh
```

Comment:
I was sure I dealt with it when I saw the shippable build error. :)

https://github.com/MRPT/mrpt/pull/556/commits/f2834a3b1bdb519d8c2718066314bc0c89dd7d5d